install/kubernetes/cilium/templates/spire/server/clusterrole.yaml

```diff
@@ -0,0 +1,102 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
```

Comment:
file name mentions clusterrole but it contains more than that

Reply:
thanks, I have split the content of this file into multiple files (e.g. role, rolebinding), which is similar to what we have in other components.